ansible/roles/software/ckan/inventory/templates/etc/ckan/production.ini

```diff
@@ -109,8 +110,12 @@ ckan.redis.url = {{ inventory_ckan_redis_url}}
 #		Add ``resource_proxy`` to enable resorce proxying and get around the
 #		same origin policy
 # ckan.plugins = usmetadata datajson datastore datapusher stats text_view recline_view googleanalyticsbasic saml2 dcat_usmetadata
+{% if inventory_next %}
+ckan.plugins = {{ (inventory_next_ckan_plugins_default + inventory_ckan_plugins_default + inventory_ckan_plugins_additional) | join(' ') }}
+{% else %}
```

Comment:
I think we'll want to close this if statement. Similar to [this](https://github.com/GSA/datagov-deploy/blob/develop/ansible/roles/software/ckan/catalog/ckan-app/templates/etc_apache2_sites-enabled_ckan.conf.j2#L32-L47)

Reply:
I was able to get rid of the if/else.